runtime/calamari/src/weights/pallet_manta_sbt.rs

```diff
@@ -64,38 +64,40 @@ impl<T: frame_system::Config> pallet_manta_sbt::WeightInfo for SubstrateWeight<T
     // Storage: MantaSbt SbtMetadata (r:0 w:1)
     // Storage: MantaSbt Shards (r:0 w:1)
     fn to_private() -> Weight {
-        (29_910_952_000 as Weight)
+        (39_461_720_000 as Weight)
```

Comment:
1/3 increase

Reply:
this one is worth thinking about if there's a code reason why this is heavier. 10ms difference is a lot @zqhxuyuan

Reply:
should be fine compare to manta-pay toPrivate: https://github.com/Manta-Network/Manta/blob/manta/runtime/calamari/src/weights/pallet_manta_pay.rs#L66

Reply:
Yeah these benchmarks are from my local machine